src/core/input_descriptor.rs

```diff
@@ -540,6 +540,7 @@ impl ConstraintsField {
         RequestedField {
             id: uuid::Uuid::new_v4(),
             name: self.name.clone(),
+            path: self.path.iter().map(|j| j.to_string()).collect(),
```

Comment:
Is it possible to select only the path that matched the requested field?

IIRC, `self.path` in this instance is the _possible_ paths where the field may reside. One of these paths should match the constraint and we should use that path.

WDYT?

Reply:
do we know that at `OID4VP`?